### PR TITLE
whitelist routes from api

### LINKED ISSUE #1643: exempt informational endpoints from monthly limit
e.g. keys, metadata.

Otherwise users who exceed the monthly rate limit are unable to do anything, including sign up for an API key.

@albertcui

### PR DESCRIPTION
**Pull request by albertcui**
fixes #1643

### CODE REVIEW

File: svc/web.js [hunk 1]
```diff
@@ -29,6 +29,16 @@ const sessOptions = {
   maxAge: 52 * 7 * 24 * 60 * 60 * 1000,
   secret: config.SESSION_SECRET,
 };
+
+const whitelistedPaths = [
+  '/api', // Docs
+  '/api/metadata', // Login status
+  '/login',
```
Comment (howardchung) [inline svc/web.js]: think passport routes are before the rate limiter anyway so it shouldn't matter

Comment (albertcui) [reply svc/web.js]: they won't get 429'd but they'll get API incremented.

File: svc/web.js [hunk 2]
```diff
@@ -29,6 +29,16 @@ const sessOptions = {
   maxAge: 52 * 7 * 24 * 60 * 60 * 1000,
   secret: config.SESSION_SECRET,
 };
+
+const whitelistedPaths = [
```
Comment (howardchung) [inline svc/web.js]: might want to refactor this into its own file and then reference it in both web and test

File: svc/web.js [hunk 3]
```diff
@@ -115,7 +125,7 @@ app.use((req, res, cb) => {
     rateLimit = config.API_KEY_PER_MIN_LIMIT;
     console.log('[KEY] %s visit %s, ip %s', requestAPIKey, req.originalUrl, ip);
   } else {
-    res.locals.usageIdentifier = `USER:${ip}:${req.user ? req.user.account_id : ''}`;
+    res.locals.usageIdentifier = `USER:${ip}`;
```
Comment (howardchung) [inline svc/web.js]: will this change effectively reset everyone's rate limits?

Comment (albertcui) [reply svc/web.js]: yes, but there's only a couple of days usage and this month is almost over anyway.

**Pull request merged**